Example/PaymentSheet Example/PaymentSheet Example/ExampleWalletButtonsView.swift

```diff
@@ -309,10 +325,11 @@ class ExampleWalletButtonsModel: ObservableObject {
                 configuration.customer = .init(id: customerId, customerSessionClientSecret: customerSessionClientSecret)
                 configuration.returnURL = "payments-example://stripe-redirect"
                 configuration.willUseWalletButtonsView = true
+                configuration.appearance = self?.appearance ?? PaymentSheet.Appearance()
```

Comment:
super nit: instead of having this logic twice with `?? PaymentSheet.Appearance()`, probably should have just updated the init to set it..